ci/cron/daily-compat.yml

```diff
@@ -23,53 +23,6 @@ schedules:
   always: true
 
 jobs:
-  - job: compatibility_ts_libs
```

Comment:
I’m confused, why is there so much deleted stuff?

Reply:
Second commit will be dropped before merging; only the first one needs review. Sorry, should have mentioned that.

Reply:
Well actually, that failed, so hold off on reviewing.